src/aai/aai.js

```diff
@@ -72,6 +72,11 @@
               case 'local':
                 // DiscoJuice.UI.setScreen(opts.localauth);
                 // jQuery('input#login').focus();
+                var expires = new Date();
+                expires.setTime(expires.getTime() + 1 * 24 * 60 * 60 * 1000);
+                var cookieString = "dsRedirectUrl=" + getCookie('dsRedirectUrl') + ";expires=" + expires.toUTCString() + ";path=/";
```

Comment:
is this really needed?

Reply:
It is working without it, I will remove it, thank you